apps/ui/src/components/Ui/Address.vue

```diff
@@ -0,0 +1,43 @@
+<script setup lang="ts">
+import { shortenAddress } from '@/helpers/utils';
+
+withDefaults(
+  defineProps<{
+    address: string;
+    copyButton?: 'always' | 'hover';
+  }>(),
+  { copyButton: 'hover' }
+);
+
+const { copy, copied } = useClipboard();
+</script>
+
+<template>
+  <span class="flex items-center gap-1.5 group">
+    {{ shortenAddress(address) }}
+    <UiTooltip
+      title="Copy address"
+      :class="{
+        'hidden group-hover:block group-focus-within:block':
+          copyButton === 'hover'
+      }"
+    >
+      <button
+        type="button"
+        class="text-skin-text h-[18px]"
+        @click.prevent.stop="copy(address)"
+        @keydown.enter.prevent.stop="copy(address)"
+        @keydown.space.prevent.stop="copy(address)"
```

Comment:
I'm wondering do we need those? I think space is non-obvious, but shouldn't enter and click be the same for accessibility?